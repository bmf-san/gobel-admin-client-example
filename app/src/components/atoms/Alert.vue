<template>
  <div
    v-if="error"
    class="alert alert-danger margin-bottom-2rem color-text-reverse"
  >
    <p>{{ error }}</p>
  </div>
</template>

<script>
import { defineComponent } from "vue";

export default defineComponent({
  name: "Alert",
  props: {
    error: {
      type: String,
      required: false,
      default: "",
    },
  },
});
</script>
