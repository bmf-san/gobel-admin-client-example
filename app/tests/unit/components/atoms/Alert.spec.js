import { shallowMount } from "@vue/test-utils";
import Alert from "@/components/atoms/Alert";

describe("Alert", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallowMount(Alert);
  });

  it("matches snapshot", () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
