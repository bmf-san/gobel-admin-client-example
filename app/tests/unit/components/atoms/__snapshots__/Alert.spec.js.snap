// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert matches snapshot 1`] = `<!--v-if-->`;
